api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -213,6 +213,10 @@ type AzureManagedControlPlaneSpec struct {
 	// OIDCIssuerProfile is the OIDC issuer profile of the Managed Cluster.
 	// +optional
 	OIDCIssuerProfile *OIDCIssuerProfile `json:"oidcIssuerProfile,omitempty"`
+
+	// This cannot be updated once the Managed Cluster has been created.
```

Comment:
```suggestion
	// DNSPrefix is ... This cannot be updated once the Managed Cluster has been created.
```
Nit: add a doc comment explaining what the field is

Reply:
Ack